Showcase-iOS/Login/Interactor/LoginInteractor.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  LoginInteractor.swift
+//  Showcase-iOS
+//
+//  Created by Lehlohonolo Mbele on 2018/04/20.
+//  Copyright © 2018 DVT. All rights reserved.
+//
+
+import Foundation
+
+class LoginInteractor: PresenterInteractable {
+    var loginPresenter: InteractorPresentable?
+    var userAuthenticator: Authenticating?
```

Comment:
how does this get init'd?

Reply:
I am using property injection to inject these.